pages/button-group/permutations.page.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import ScreenshotArea from '../utils/screenshot-area';
+import PermutationsView from '../utils/permutations-view';
+import createPermutations from '../utils/permutations';
+import { ButtonGroup, ButtonGroupProps } from '~components';
+
+const permutations = createPermutations<ButtonGroupProps>([
+  {
+    variant: ['icon'],
+    dropdownExpandToViewport: [true, false],
+    items: [
```

Comment:
I would create a few separate arrays of items and create permutations with different items. The things to check are e.g.: item-only button group, button group with adjacent groups, with a group separated with actions on both sides, with one show-more dropdown, with multiple show-more dropdowns, with show-more dropdown inside a group, etc.